docs/design/openshift.md

```diff
@@ -96,15 +96,16 @@ The policy engine client API conforms to the [Cincinnati Graph API](cincinnati.m
 
 #### Request ####
 
-HTTP GET requests are used to fetch updates from the policy engine. Requests are made to `/graph` and must include the following URL parameters:
+HTTP GET requests are used to fetch updates from the policy engine. Requests are made to `/graph` and include the following URL parameters:
```

Comment:
maybe "must" -> "support" or something that that allows for not including the optional params?

Reply:
Revised.
Please let me know if does not look good enough still.

Reply:
7e1f438f6078234b59's `with` looks good to me; I'm marking this thread resolved.